dev-docs/creating-a-worker-ami.md

```diff
@@ -0,0 +1,140 @@
+# Creating a worker AMI for your Taskcluster deployment
+
+Once you have your Taskcluster deployment up and running, you might want to look at getting workers setup.
+This guide will walk you through how to use [monopacker](https://github.com/taskcluster/monopacker) to automatically build an AWS AMI for your workers.
+
+## Prerequisites
+
+You will need to have the following
+
+* An AWS account with an IAM user with an API key.
+* Some funds on your AWS account, or backed by a creditcard: building, using and even storing the AMIs you something you will get billed for.
+
+### Cloning the repository and isntalling.
+
+First you will want to clone the [monopacker](https://github.com/taskcluster/monopacker) repository.
+Set up a folder and run `git clone https://github.com/taskcluster/monopacker` and then `cd` into the directory.
+
+Make sure you have python3 and python3-venv installed. Then run `python3 -m venv dev` to make a python virtual environment in the dev directory. Run `source ./dev/bin/activate` to activate it.
+
+To install everything we need, from within the venv, run `python setup.py install`.
+
+### Configuring the template
+
+First we edit `template/builders/amazon_ebs.jinja2` and under `ami_users` set your AWS account ID.
+
+Then we edit `template/vars/default_aws.yaml` and we change the following:
+
+* Change `instance_type` to an instance type that has the amount of capacity that you will need. For example `c5.large`.
+* Change `region:` to the region that you want to build the AMI in.
+* Change `ami_regions:` to regions where you wish to copy the AMI to and use it to deploy workers.
+* Change `volume_size:` to the volume size that you want your boot volume to be, in GB.
```

Comment:
Did you end up increasing or decreasing this?  Any particular reason to do so?

Reply:
I changed the ami_regions to only be one region (for cost savings), same for volume_size, I made it 20GB, for cost savings.